framework/templates/ecl-templates-listings/ecl-templates-listings.twig

```diff
@@ -290,6 +290,38 @@
         },
       ]
     } %}
+
+    <h3 class="ecl-heading ecl-heading--h3">Highlight listing (three columns)</h3>
+
+    {% include '@ec-europa/ecl-listings' with {
+      variant: 'three-columns',
+      items: [
+        {
+          variant: 'highlight',
```

Comment:
A `screen_reader` would be useful containing the title rather than the default.

Reply:
indeed